estela-web/src/components/JobDetailPage/index.tsx

```diff
@@ -234,7 +234,7 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
             pid: this.projectId,
             sid: this.spiderId,
             jid: this.newJobId,
-            type: "stats",
+            type: "logs",
         };
         this.apiService.apiProjectsSpidersJobsDataList(requestParams).then(
```

Comment:
We are missing a button to see the job logs like the ones we have for items and requests.

Reply:
It was decided to leave the entrypoint alone for now.